### PR TITLE
DSPDC-679 Consolidate Helm releases into a single chart, using Flux

### PR DESCRIPTION
**Pull request by danxmoran**
I think I like it now that it's working, but I'm curious to hear what others think!

### CODE REVIEW

File: environments/dev/terraform/variables.tf [hunk 1]
```diff
@@ -1,9 +1,3 @@
-variable kubeconfig_dir_path {
-  type = string
-  description = "Local path where all kubeconfigs generated in the environment should be written."
-}
-
 locals {
-  processing_kubeconfig_dir = "${var.kubeconfig_dir_path}/processing"
   vault_prefix = "secret/dsde/monster/dev"
```
Comment (quazi-broad) [inline environments/dev/terraform/variables.tf]: Can this variable be referenced from the common script?
I noticed you had to specify the vault path there, too.

File: hack/apply-terraform [hunk 2]
```diff
@@ -43,10 +44,14 @@ function main () {
...
     ${TERRAFORM}
   )
+
+  # Clear out any cached local module state.
+  rm -rf ${env_dir}/terraform/.terraform/modules
+
   ${terraform[@]} init
   # Hard-coding kubeconfig path for now because there's a ticket to
   # stop writing it to local disk at all.
-  ${terraform[@]} apply -var="kubeconfig_dir_path=/env/.kubeconfig"
```
Comment (snwessel) [inline hack/apply-terraform]: It looks like you removed the hard-coded kubeconfig path, so maybe remove the comment above as well

File: templates/helm/command-center/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v2
+name: command-center
+description: Services used to run the "command center" in a Monster environment.
+version: 0.1.0
+dependencies:
+  - name: crd-operators
+    version: 0.1.0
+    repository: file:///charts/crd-operators
```
[the remark this answers is not in the document]
Comment (danxmoran) [reply templates/helm/command-center/Chart.yaml]: `file://` is the scheme (i.e. `http://`). The 3rd `/` is the start of the path.

File: environments/dev/terraform/variables.tf (hunk 1 above)
Comment (danxmoran) [reply environments/dev/terraform/variables.tf]: I'd like to wait until we instantiate our prod environment to see where it makes sense to parameterize this. I'm sure there are ways we could do it, but I'm not sure which makes the most sense right now.

**Pull request merged**